Initial draft of SIG-Usability.

brendandburns opened the pull request:
This is an initial rough draft for discussion, lots of details need to be filled in.

vllry commented:
/cc

asauber commented on sig-usability/charter.md:
```diff
@@ -0,0 +1,52 @@
+
+* Contributor user experience (covered by SIG-Contributor Experience)
+* API Design (covered by SIG-Architecture)
+* Command line tool infrastructure (covered by SIG-CLI)
```
I would expect that this SIG make efforts towards improving the command line tools' input and output interfaces.

For example, the addition of `-A` has saved me countless keystrokes since its introduction.

vllry replied:
Perhaps "advisory capacity" for those topics? I agree those sigs should own those components, but it would be good to codify the subject-authority of the SIG.

jdumars replied:
SIGs only have authority over their own sub-projects, and are limited to influence on related topics, so "advisory capacity" makes sense. This SIG should be actively identifying areas of the project where they can influence decision-making to include/increase accessibility. This is similar to contribx where they ensure the contributor is considered in process and other changes.

BenTheElder replied:
Also not all CLI tools are even SIG-CLI necessarily, e.g. minikube.

However, I'd love to see _someone_ own common tooling (as opposed to kubectl internal) for our CLIs for these problems. More concretely I'd like to see tooling we can use for [kind](https://github.com/kubernetes-sigs/kind) at some point...
I would hope this would be in-scope for a SIG.

brendandburns replied:
I added some of this advisory stuff in the section above.

@BenTheElder I think that general CLI tooling libraries, etc. are out of scope for this SIG, though clearly useful for someone to produce :)

pwittrock replied:
@BenTheElder Depending on what you are envisioning, https://github.com/kubernetes/cli-runtime/ could be a place to put shared libraries for developing clis.

cblecker commented:
/hold
/cc @kubernetes/steering-committee 
/committee steering

Holding for steering committee review/approval :)

asauber commented on sig-usability/charter.md:
```diff
@@ -0,0 +1,52 @@
+- accessibility of Kubernetes community artifacts, examples include:
+   - internationalization of documentation in binaries and documentation
+   - color choices for people with color blind-ness
+   - ensuring compatability with screen reader technology
```
Screen reader friendliness also applies in the context of terminal interfaces.

brendandburns replied:
Not sure I 100% follow.

The existing statement is trying to cover that we would consider this for terminal tools like `kubectl` is there something I could add here to make that more clear?

asauber replied:
I understood "community" artifacts to not include CLI binaries (rather, website and other documentation). However, I could have seen "binaries" mentioned in the previous bullet point.

vllry commented:
Regarding SIG leadership, I'd like to see some social/technical authority within the community, to help make sure the SIG is productive and listened to within the development community. I'm happy to volunteer, but I will gladly step back if someone more qualified would like to step up.

jdumars commented on sig-usability/README.md:
```diff
@@ -0,0 +1,47 @@
+
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+--->
+# Usability Committee
```
nit: from the governance perspective, this isn't a committee. s/group/committee/g

brendandburns replied:
@jdumars I think this came from the generator?

brendandburns replied:
I think I found the problem, I put this in the wrong list, fixed.

philips commented:
I agree with the goals but I wonder if we need to have a working group come together first and figure out how to practically review error codes across the code base and internationalization. And then generate a KEP.

As of today this SIG would essentially own no code, right?

brendandburns commented:
@philips I don't think a working group is the right idea, since a working group has a fixed outcome. What would that outcome be?

I think that the SIG would have exclusive ownership of like the internationalization code. And would have advisorial responsibility for much more of the code.

I don't think SIGs exclusively owning code is a requirement, is it? (in that case there are several SIGs that don't meet that bar)

I think that there is clearly sufficient user interest, and I think the scope is there for a SIG (and the ownership of code within Kubernetes is also there, even if it's not exclusive)

philips commented:
@brendanburns I think the working group would create a prototype and KEP for how to do internationalization and error code management technically. It would be a lightweight way of getting interested people working together as well.

I don't have a strong stance on this, just a feeling that creating a SIG without a prototype and proposal might be premature. But, you mentioned "I think that there is clearly sufficient user interest, " so I might be out of the loop.

brendandburns commented:
@philips there's a thread on sig-architecture.

And we already prototyped internationalization in the CLI a long time ago. I think what's needed is to get people to gather and focus on it.

jdumars commented:
@philips for reference: https://groups.google.com/d/msg/kubernetes-sig-architecture/Wp7V9KTrB5c/BR4Ex3j8BAAJ

philips commented:
I think including links to those existing internationalization efforts and
the context on the sig-architecture list would be helpful for people coming
into this without context from the steering committee. Thanks!

FWIW, I am OK with a SIG, just brainstorming on the WG thing and I saw it
was already discussed on the sig-architecture thread.

On Fri, Jun 14, 2019 at 1:54 PM Jaice Singer DuMars <
notifications@github.com> wrote:

> @philips <https://github.com/philips> for reference:
> https://groups.google.com/d/msg/kubernetes-sig-architecture/Wp7V9KTrB5c/BR4Ex3j8BAAJ
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/community/pull/3801?email_source=notifications&email_token=AAAIGCAOHNPP2QXAXHEIVX3P2QAPBA5CNFSM4HYL5FX2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODXX6AAA#issuecomment-502259712>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAIGCAUEVWWK2H6FSJTOCDP2QAPBANCNFSM4HYL5FXQ>
> .
>

brendandburns commented:
@philips here's the internationalization code:

https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/util/i18n/i18n.go
Also
https://github.com/kubernetes/kubernetes/tree/master/translations

If you set `LANG` environment variable to a supported language (say `export LANG=fr_FR`) and then run kubectl help you should see some translations. Not everything in the CLI is supported, one of the tasks for the SIG would be to make this automatic/required for new flags and messages.

I updated the charter with these locations.

BenTheElder commented:
A bit of an aside but: That the i18n work above seems to be kubectl specific; I'm not sure where other SIG tools are at on this but it might be helpful to push this out to other tooling. Producing reusable tools / libraries and guidance for this could be really helpful for tools within the project beyond kubectl and seems like something this WG / SIG would own... 🙃

brendandburns commented:
@BenTheElder absolutely, especially since a lot of this moved to the server, I'd like to see us move internationalization beyond kubectl (though also, please note that the intended scope is def. beyond i18n :)

mattfarina commented on sig-usability/README.md:
```diff
@@ -0,0 +1,50 @@
+--->
+# Usability Special Interest Group
+
+The scope of SIG usability is the core end-user usability of the Kubernetes project. Thiis covers topics like user experience and accessibility. The goals of the SIG are to ensure that the Kubernetes project is maximaly usable by users of a variety of different backgrounds and abilities. Examples of efforts include internationalization and accessibility of documentation.
```
"Thiis" -> "This"

brendandburns replied:
Done.

jperitz replied:
```suggestion
The scope of SIG usability is the core end-user usability of the Kubernetes project. Thiis covers topics like user experience and accessibility. The goals of the SIG are to ensure that the Kubernetes project is maximaly usable by users of a variety of different backgrounds and abilities. Examples of efforts include internationalization and accessibility of documentation.
```

jperitz replied:
Also "maximaly" -> "maximally"

brendandburns replied:
done.

mattfarina commented on sig-usability/charter.md:
```diff
@@ -0,0 +1,54 @@
+
+## Scope
+
+The scope of SIG usability is the core end-user usability of the Kubernetes project. Thiis covers
```
"Thiis" -> "This"

brendandburns replied:
done.

mattfarina commented on sig-usability/charter.md:
```diff
@@ -0,0 +1,54 @@
+
+#### Cross-cutting and Externally Facing Processes
+
+- None currently, eventually perhaps a more formal usability review for changes
```
Usability is typically a "cross-cutting" concern. A lot of what they can provide is information for other SIGs to build more useful stuff. useful = usability + utility.

From an "externally facing" standpoint, the CNCF end user community is looking to engage with projects.  Is this something SIG Usability should formally engage in?

justaugustus replied:
@mattfarina -- I'm going to be working with @oicheryl to formalize CNCF EUC Comms as a subproject of SIG PM.

gaby-moreno replied:
Suggestion:
#### Cross-cutting and Externally Facing Processes

SIG Usability will facilitate and collaborate with other SIGs on 
- giving end users channels for proving feedback on their experience
- documenting common end user profiles (aka personas)
- documenting common end user interactions with Kubernetes (aka user journeys)
- identifying opportunities for user research and best methods to achieve (qualitative and quantitative).

brendandburns replied:
Done.

mattfarina commented on sig-usability/charter.md:
```diff
@@ -0,0 +1,54 @@
+   - ensuring compatibility with screen reader technology
+- user interface design for core components with user interfaces.
+- translations - https://github.com/kubernetes/kubernetes/tree/master/translations
+- internationalization - https://github.com/kubernetes/kubernetes/tree/master/pkg/kubectl/util/i18n
```
The list of code, binaries, and services highlights areas we are often concerned with. While I wouldn't expect SIG Usability to own any of these, I would expect them to own UX profiles or personas (as the material gets mature enough to do personas) and some other usability meta things. How should we capture that?

gaby-moreno replied:
Suggestion:
### In scope

#### Repos
Suggest we create a separate user research repo for documentation of:
- Common end user profiles (aka personas)
- Ongoing list of research studies and findings
- Ongoing list of known needs for user research (ex: top stackoverflow questions)
- Curated research on usability best practices (ex: HCI research literature on interface usability)
- Templates for anybody to be able to conduct user research interviews

Suggest we create a separate accessibility repo for documentation of:
the bullet points already mentioned

brendandburns replied:
done.

mattfarina commented:
This feels like a coder wrote the scope and charter for the usability SIG. Could we get some usability people to take a shot at refining it?

gaby-moreno commented:
> This feels like a coder wrote the scope and charter for the usability SIG. Could we get some usability people to take a shot at refining it?

happy to take a look, we tried submitting a kubecon birds of a feather on this topic of a sig ux which didn't make it in. super excited to see it getting traction!

gaby-moreno commented on sigs.yaml:
```diff
@@ -2205,6 +2205,53 @@ sigs:
+    are to ensure that the Kubernetes project is maximaly usable by users of a variety
+    of different backgrounds and abilities. Examples of efforts include internationalization
+    and accessibility of documentation.
+
```
I like this mission statement but feel like making something _usable_ does not mean making it a *positive experience* that encourages trust and continued use. 

Suggestion: "The goal of SIG "Usability" is to facilitate adoption of the Kubernetes project by as diverse a community of end users as possible. We do this be ensuring that each end user’s interaction with Kubernetes, from discovery to successful production use is seamless and positive. Examples of efforts include user research, internationalization and accessibility."

Still throwing out SIG End User Experience as a name suggestion since that makes the connection to our CNCF End User community more explicit. Echoing other people's assumptions that this SIG would be responsible for collaborating with other SIGs to give the end users channels for providing feedback on their experience.

gaby-moreno commented on sig-usability/charter.md:
```diff
@@ -0,0 +1,54 @@
+   - internationalization of documentation in binaries and documentation
+   - color choices for people with color blind-ness
+   - ensuring compatibility with screen reader technology
+- user interface design for core components with user interfaces.
```
I would think that user interface design for core components or any design system in general would fall under the realm of SIG UI and that research from SIG Usability would inform that.

brendandburns replied:
integrated.

gaby-moreno commented on sig-usability/charter.md:
```diff
@@ -0,0 +1,54 @@
+
+### Deviations from [sig-governance]
+
+No Tech Leads (for now)
```
It would be great to have Tech Leads as part of SIG Usability that can collaborate with ux researchers to make sure that ongoing research with end users has the depth it needs to be useful to the community.

brendandburns replied:
Happy to add people as part of growing the SIG, but I think that we need to grow tech. leads organically through the operation of the SIG rather than identify them as part of formation.

vllry commented:
The initial idea is less than 48 hours old, and we already have no end of suggestions for responsibilities:

* Accessibility
* Documentation
* General user-experience advice
* End-user relationship management
* Internationalization
* User research
* User research frameworks

On one hand, it's very exiting to have this much engagement! It definitely speaks to gaps that need to be addressed. Personally I'd been mulling over a "sig-operability" proposal for quite some time.

On the other hand, I'm worried that this is falling prey to massive scope creep to the point of inefficacy. The topics suggested are all **very*** deep and involved. We all want these things to be addressed, but realistically, we will have limited people and limited time. Additionally, there is heavy overlap between some suggestions and existing groups (such as sig-docs, which is already supposed to fully own documentation accuracy/usefulness/strategy).

My humble suggestion is that we avoid an "aspiration" responsibility set, and focus on the core "be the champions and eyes in the sky for user experience", as both (a) topic experts and (b) cross SIG coordinators, similar to the role of SIG-architecture in high-scope technical decisions. So, something along the lines of "gather holistic user feedback, and use it to assist individual SIGs in their designs and prioritization". Subjects like common internationalization tooling would be a fantastic evolution of scope in the longer term. I'll submit some suggested text shortly, to be a little more helpful than "I have concerns". 🙂

(Or, we could go with the approach of creating a charter-forming WG as @philips suggested 🙃)

vllry commented:
Another possibility would be creating +1 subprojects under an existing SIG (probably PM). UX and user research is an attractive surface for some backseat roadmap steering from vendors.

Rajakavitha1 commented:
> I was thinking this should include something about user research, persona definition, understanding the new user on-boarding experience for those defined personas and optimizing it.
> 
> collaborate to create uniform open sourced design elements (e.g. icons) for different cloud native concepts, adding on to the clarity project https://clarity.design/ ,
> 
> Tracking adherence to required guidelines such as WCAG (required to sell products in Europe): https://www.w3.org/WAI/news/2018-09-13/WCAG-21-EN301549/
> 
> And consider creating a site in the vein of https://www.microsoft.com/en-us/accessibility/

I would also like to add : https://www.w3.org/WAI/standards-guidelines/wcag/ . SID-Docs created the writing guidelines based on this.

Rajakavitha1 commented on sig-list.md:
```diff
@@ -49,6 +49,7 @@ When the need arises, a [new SIG can be created](sig-wg-lifecycle.md)
 |[Storage](sig-storage/README.md)|storage|* [Bradley Childs](https://github.com/childsb), Red Hat<br>* [Saad Ali](https://github.com/saad-ali), Google<br>|* [Slack](https://kubernetes.slack.com/messages/sig-storage)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-storage)|* Regular SIG Meeting: [Thursdays at 9:00 PT (Pacific Time) (biweekly)](https://docs.google.com/doc ...
 |[Testing](sig-testing/README.md)|testing|* [Erick Fejta](https://github.com/fejta), Google<br>* [Aaron Crickenberger](https://github.com/spiffxp), Google<br>* [Steve Kuznetsov](https://github.com/stevekuznetsov), Red Hat<br>* [Timothy St. Clair](https://github.com/timothysc), VMware<br>|* [Slack](https://kubernetes.slack.com/messages/sig-testing)<br>* [Mailing List](https://groups.google.com/for ...
 |[UI](sig-ui/README.md)|ui|* [Dan Romlein](https://github.com/danielromlein), Google<br>* [Sebastian Florek](https://github.com/floreks), Loodse<br>* [Jeffrey Sica](https://github.com/jeefy), University of Michigan<br>* [Marcin Maciaszczyk](https://github.com/maciaszczykm), Loodse<br>|* [Slack](https://kubernetes.slack.com/messages/sig-ui)<br>* [Mailing List](https://groups.google.com/forum/#!for ...
+|[Usability](sig-usability/README.md)|usability|* [Rajakavitha Kodhandapani](https://github.com/unknown), unknown<br>* [Vallery Lancey](https://github.com/unknown), unknown<br>* [Himanshu Pandey](https://github.com/unknown), unknown<br>|* [Slack](https://kubernetes.slack.com/messages/sig-usability)<br>* [Mailing List](TBD)|* Regular SIG Meeting: [TBDs at TBD PT (Pacific Time) (every third week)]( ...
```
[Rajakavitha Kodhandapani](https://github.com/Rajakavitha1)

brendandburns replied:
If it's ok with you, I'd prefer to leave you as a lead. 

I have no particularly strong need to be chair, and I'd rather empower others in the community. I'll certainly help get this thing started as it is an area of interest for me.

tashimi replied:
I'd be interested in being a lead.

tashimi replied:
Also I think @gaby-moreno would be an excellent lead

brendandburns replied:
If folks want to re-arrange as we form up, I think that's great. SIG chair can be a bunch of work, so starting out with a larger number of chairs and then reducing down or rotating if there's not enough work seems like a good plan. 

@tashimi I added you.

@gaby-moreno if you want me to add you as a chair let me know.

gaby-moreno replied:
@brendanburns would love to, I also agree with your comment above.

parispittman commented:
could this be a group that could also own end user communication infrastructure and the experience of those folks on them - ie: slack?

gaby-moreno commented:
> could this be a group that could also own end user communication infrastructure and the experience of those folks on them - ie: slack?

I would think that would be a core competency of this group!

vllry commented:
These suggestions are starting to diverge in 2 themes - user experience and user outreach.

brendandburns commented:
I'm w/ @vllry I don't want to scope creep to everything. Let's stay focused on user experience/traditional usability for now.

Folks, I've added/integrated most comments, please let me know if there is additional feedback.

Thanks!

zacharysarah commented:
Hi, folks--I just got back from an extended vacation. 👋 I'll read through things more carefully over the course of the week, but some initial impressions:
- Yay engagement for usability and accessibility!
- I'm concerned by talk about new locations of documentation, but I see no invitations to or participation from SIG Docs. Is initializing this SIG sufficiently urgent to shortcut that discussion?

brendandburns commented:
@zacharysarah I don't think any of the documentation is intended for external users. It is things like "document how to conduct a user research study"

I view that as the same as "document how to run a e2e tests" (https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md) which aren't generally part of SIG-Docs afaik.

We don't have any desire to overlap with SIG-Docs, and if SIG-Docs folks have any concerns, this is PR is the right forum to raise any concerns, or ask for clarifications for charter.

Ultimately, whether the SIG should exist is a steering committee decision.

Once we get consensus/clarity on the charter, we'll be taking it to the steering committee for approval (and clearly I will recuse myself...)

brendandburns commented:
I added @gaby-moreno as a lead, fixed a typo.

Are we reaching quiescence?

I'm starting to drive the steering committee conversation to get this approved.

If there is other feedback that people want in this initial charter, remembering that charters are living documents that are intended to evolve, please comment asap.

vllry commented:
> Are we reaching quiescence?

I'm good as it stands!

zacharysarah commented on sig-usability/README.md:
```diff
@@ -0,0 +1,52 @@
+--->
+# Usability Special Interest Group
+
+The scope of SIG usability is the core end-user usability of the Kubernetes project. Thiis covers topics like user experience and accessibility. The goals of the SIG are to ensure that the Kubernetes project is maximally usable by users of a variety of different backgrounds and abilities. Examples of efforts include internationalization and accessibility of documentation.
```
s/Thiis/this

zacharysarah replied:
> Examples of efforts include internationalization and accessibility of documentation.

I'm glad to see internationalization get more visibility. Because projects use a variety of static site generators, and not just Hugo like k/website, it's a good idea to aggregate i18n information across multiple projects. 

I have a couple of concerns about where to keep information about accessibility: 

1. Info about docs accessibility needs to live in the [docs contributors guide](https://kubernetes.io/docs/contribute/start/), not in a separate silo. Maintaining dual sources of information usually leads to divergence and staleness.

2. I can, however, see a case for general accessibility guidelines, similar to general i18n guidelines. 

Whatever we do with accessibility, I want to make sure that we aren't maintaining dual sources of information. That's setting contributors up for failure.

brendandburns replied:
I added a note that how-to guides for accessible documentation were out of scope and covered by SIG-docs.

zacharysarah commented on sig-usability/charter.md:
```diff
@@ -0,0 +1,71 @@
+## Scope
+
+The scope of SIG usability is the core end-user usability of the Kubernetes project. This covers
+topics like user experience and accessibility. The goal of SIG "Usability" is to facilitate adoption of
```
s/"Usability"/Usability

brendandburns replied:
done.

zacharysarah commented:
> Are we reaching quiescence?

I left a handful of comments. I support this SIG and its goals, and I want to make sure we're setting this team, our developers, and our contributors up for long-term success. When it comes to accessibility information, I foresee some avoidable problems with dual source maintenance and information silos. I think we need clarity about where accessibility information lives; if we can achieve clarity about where accessibility information lives, then yes, I will lapse into quiescence. :bowtie:

hpandeycodeit commented on sig-usability/README.md:
```diff
@@ -0,0 +1,52 @@
+* Rajakavitha Kodhandapani (**[@Rajakavitha1](https://github.com/Rajakavitha1)**), unknown
+* Gaby Moreno (**[@gaby-moreno](https://github.com/gaby-moreno)**), IBM
+* Tasha Drew (**[@tashimi](https://github.com/tashimi)**), VMWare
+* Himanshu Pandey (**[@unknown](https://github.com/unknown)**), unknown
```
@brendandburns https://github.com/hpandeycodeit
Is there any doc that lists down Roles and responsibilities for a Chair?

brendandburns replied:
See the notes here:

https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md#chair

dims commented on sig-usability/README.md:
```diff
@@ -0,0 +1,52 @@
+## Leadership
+
+### Chairs
+The Chairs of the SIG run operations and processes governing the SIG.
```
Is everyone part of the k8s org, if not, please propose them?

brendandburns replied:
@gaby-moreno @vllry @hpandeycodeit @tashimi @Rajakavitha1 

Can you confirm that you are all already members of the Kubernetes organization? If you are not, and you think you qualify for nomination, please let me know and we can start to push that forward. Otherwise, I think we'll just have you be informal chairs for now, and then you can use your participation in the SIG to become a full fledged member of the organization.

Rajakavitha1 replied:
Yes, I am a member of the Kubernetes organization.

vllry replied:
I'm a member, as is @tashimi and @hpandeycodeit. It looks like @gaby-moreno is not.

gaby-moreno replied:
Hi @brendanburns, I am not a member yet

tashimi replied:
Yeah, I'm a member. :)

brendandburns replied:
@gaby-moreno

For now, I've removed you from the list of chairs, not because we don't want you to be a chair, but just because it will be easier to form this up and then add you later.

Please review the requirements for membership here:
https://github.com/kubernetes/community/blob/master/community-membership.md#requirements

If you think that you have met the requirements, please find two sponsors (I'm happy to be one of them) and we'll get you into the org.

Otherwise, active participation in this SIG will certainly qualify you for membership in the org.

Please let me know if there are questions! Thanks!

gaby-moreno replied:
Hi @brendanburns, that's fine. Thank you for the resource and for offering to sponsor!

Rajakavitha1 commented on sig-usability/README.md:
```diff
@@ -0,0 +1,52 @@
+### Chairs
+The Chairs of the SIG run operations and processes governing the SIG.
+
+* Rajakavitha Kodhandapani (**[@Rajakavitha1](https://github.com/Rajakavitha1)**), unknown
```
The Linux Foundation (Contract)

brendandburns replied:
done.

vllry commented on sig-usability/README.md:
```diff
@@ -0,0 +1,52 @@
+* Gaby Moreno (**[@gaby-moreno](https://github.com/gaby-moreno)**), IBM
+* Tasha Drew (**[@tashimi](https://github.com/tashimi)**), VMWare
+* Himanshu Pandey (**[@unknown](https://github.com/unknown)**), unknown
+* Vallery Lancey (**[@vllry](https://github.com/vllry)**), unknown
```
* Vallery Lancey (**[@vllry](https://github.com/vllry)**), Lyft

brendandburns replied:
done.

vllry commented on sig-list.md:
```diff
@@ -49,6 +49,7 @@ When the need arises, a [new SIG can be created](sig-wg-lifecycle.md)
 |[Storage](sig-storage/README.md)|storage|* [Bradley Childs](https://github.com/childsb), Red Hat<br>* [Saad Ali](https://github.com/saad-ali), Google<br>|* [Slack](https://kubernetes.slack.com/messages/sig-storage)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-storage)|* Regular SIG Meeting: [Thursdays at 9:00 PT (Pacific Time) (biweekly)](https://docs.google.com/doc ...
 |[Testing](sig-testing/README.md)|testing|* [Erick Fejta](https://github.com/fejta), Google<br>* [Aaron Crickenberger](https://github.com/spiffxp), Google<br>* [Steve Kuznetsov](https://github.com/stevekuznetsov), Red Hat<br>* [Timothy St. Clair](https://github.com/timothysc), VMware<br>|* [Slack](https://kubernetes.slack.com/messages/sig-testing)<br>* [Mailing List](https://groups.google.com/for ...
 |[UI](sig-ui/README.md)|ui|* [Dan Romlein](https://github.com/danielromlein), Google<br>* [Sebastian Florek](https://github.com/floreks), Loodse<br>* [Jeffrey Sica](https://github.com/jeefy), University of Michigan<br>* [Marcin Maciaszczyk](https://github.com/maciaszczykm), Loodse<br>|* [Slack](https://kubernetes.slack.com/messages/sig-ui)<br>* [Mailing List](https://groups.google.com/forum/#!for ...
+|[Usability](sig-usability/README.md)|usability|* [Rajakavitha Kodhandapani](https://github.com/Rajakavitha1), The Linux Foundation<br>* [Gaby Moreno](https://github.com/gaby-moreno), IBM<br>* [Tasha Drew](https://github.com/tashimi), VMWare<br>* [Himanshu Pandey](https://github.com/unknown), unknown<br>* [Vallery Lancey](https://github.com/vllry), unknown<br>|* [Slack](https://kubernetes.slack.c ...
```
|[Usability](sig-usability/README.md)|usability|* [Rajakavitha Kodhandapani](https://github.com/Rajakavitha1), The Linux Foundation<br>* [Gaby Moreno](https://github.com/gaby-moreno), IBM<br>* [Tasha Drew](https://github.com/tashimi), VMWare<br>* [Himanshu Pandey](https://github.com/unknown), unknown<br>* [Vallery Lancey](https://github.com/vllry), Lyft<br>|* [Slack](https://kubernetes.slack.com/messages/sig-usability)<br>* [Mailing List](TBD)|* Regular SIG Meeting: [TBDs at TBD PT (Pacific Time) (every third week)](tbd)<br>

brendandburns replied:
done.

hpandeycodeit replied:
@brendandburns `company: Pivotal`

vllry commented on sigs.yaml:
```diff
@@ -2208,6 +2208,59 @@ sigs:
+      company: unknown
+    - github: vllry
+      name: Vallery Lancey
+      company: unknown
```
company: Lyft

brendandburns replied:
done.

brendandburns commented:
Comments all addressed, except need Kubernetes org membership info for potential chairs to address @dims

brendandburns commented:
Comments addressed. Any more feedback?

I'm going to start pushing this with the steering committee to get see if we can get approval.

dims commented:
LGTM 👍

frapposelli commented on sigs.yaml:
```diff
@@ -2208,6 +2208,56 @@ sigs:
+      company: Pivotal?
+    - github: tashimi
+      name: Tasha Drew
+      company: VMWare
```
```suggestion
      company: VMware
```

pwittrock commented:
/lgtm

dims commented:
/lgtm

philips commented:
/lgtm

brendandburns commented:
Can someone re-add LGTM, this was just a rebase.

brendandburns commented:
This has been discussed and approved via lazy consensus by the Kubernetes steering committee.

/remove-hold

brendandburns commented:
I mean:

/hold cancel

justaugustus commented:
/lgtm
/approve

brendandburns commented:
w00t!

@vllry @gaby-moreno @Rajakavitha1 @tashimi 

We need to create a mailing list, slack channel and select a meeting time.

Who wants to own what?

vllry commented:
I'll look into Slack and mailing lists, I think we'll need to go back and forth a bit around meeting cadence and times (which will be a lot easier with comms).

brendandburns commented:
I suggest an every 3 weeks cadence, but let's use a doodle to find the optimum time/frequency....

tashimi commented:
Sounds good - is first step to draft a charter? https://github.com/kubernetes/community/tree/master/committee-steering/governance

tashimi commented:
Doodle for scheduling the first meeting: https://doodle.com/poll/8a282ky76rhz7diu

brendandburns commented:
@vlry see the mailing list guidelines here:

https://github.com/kubernetes/community/blob/master/communication/mailing-list-guidelines.md

vllry commented:
Created https://groups.google.com/forum/#!forum/kubernetes-sig-usability and https://groups.google.com/forum/#!forum/kubernetes-sig-usability-leads , still need to double check the settings & set up Slack.

hpandeycodeit commented:
I cannot access the above groups. Do the permissions still need to be assigned?

vllry commented:
@hpandeycodeit I need your email address to add you to the group owners... but you should be able to join the kubernetes-sig-usability group just fine as-is. 🤔 I tried in incognito, and it worked fine for me.

Rajakavitha1 commented:
Hi,

I dont seem to access to these groups as well. My email ID is
krajakavitha@gmail.com.

Rajie

On Thu, Jul 18, 2019 at 10:29 AM Vallery Lancey <notifications@github.com>
wrote:

> @hpandeycodeit <https://github.com/hpandeycodeit> I need your email
> address to add you to the group owners... but you should be able to join
> the kubernetes-sig-usability group just fine as-is. 🤔 I tried in
> incognito, and it worked fine for me.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/community/pull/3801?email_source=notifications&email_token=AI2XZJN63BMSUJ7DWBPKQJLP772DPA5CNFSM4HYL5FX2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD2HKMZA#issuecomment-512665188>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AI2XZJMSYHAZGUJJE4MGJBLP772DPANCNFSM4HYL5FXQ>
> .
>

gaby-moreno commented:
How exciting!!! 

(I just tried to join the kubernetes-sig-usability group and also got a permissions error.)

shrasool commented:
The group join link seems to work fine now. Just joined!

oceanicsdotio commented:
Excited to dig in on this much needed SIG.
